Tasks/VSMobileCenterTest/task.json

```diff
@@ -384,6 +405,7 @@
         },
         {
             "name": "async",
+            "aliases": [ "waitForResults" ],
```

Comment:
The meaning of the alias does not match the meaning of the label. Seems like this be "skipWaitingForResults" or something similar.